packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -179,5 +179,3 @@ export class MenuToggleBase extends React.Component<MenuToggleProps> {
 export const MenuToggle = React.forwardRef((props: MenuToggleProps, ref: React.Ref<MenuToggleElement>) => (
   <MenuToggleBase innerRef={ref} {...props} />
 ));
-
-MenuToggle.displayName = 'MenuToggle';
```

Comment:
Is there inconsistency between MenuToggle and other components (TextInput as an example)? TextInput has the eslint-disable comment but doesn't have the `TextInput.displayName` line removed like here.